src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -111,61 +112,69 @@
     <desc>DATM CO2 time series</desc>
   </entry>
 
+  <entry id="DATM_CPLHIST_DOMAIN_FILE">
+    <type>char</type>
+    <valid_values></valid_values>
+    <default_value>null</default_value>
+    <group>run_component_datm</group>
+    <file>env_run.xml</file>
+    <desc>
+      Full pathname for domain file for datm when DATM model is CPLHIST, NOTE: if this is set to 'null',
+      then domain information is read in from the first coupler history file in the target stream and
+      it is assumed that the first coupler stream file that is pointed to contains the domain
+      information for that stream. This is the default mode that should be used when DATM_MODE is CPLHIST.
+    </desc>
+  </entry>
+
   <entry id="DATM_CPLHIST_DIR">
     <type>char</type>
     <valid_values></valid_values>
     <default_value>UNSET</default_value>
     <group>run_component_datm</group>
     <file>env_run.xml</file>
-    <desc>directory for coupler history data mode (only used for CPLHIST3HrWx mode)</desc>
+    <desc>directory for coupler history data mode (only used for when DATM_MODE is CPLHIST)</desc>
   </entry>
 
   <entry id="DATM_CPLHIST_CASE">
     <type>char</type>
     <valid_values></valid_values>
     <default_value>UNSET</default_value>
     <values match="last">
-      <value compset="1850_DATM%CPLHIST">b40.1850.track1.1deg.006a</value>
+      <value compset="1850_DATM%CPLHIST">UNSET</value>
     </values>
     <group>run_component_datm</group>
     <file>env_run.xml</file>
-    <desc>case name for coupler history data mode (only used for CPLHIST3HrWx mode)</desc>
+    <desc>case name used to determine stream filenames when DATM_MODE is CPLHIST</desc>
   </entry>
 
   <entry id="DATM_CPLHIST_YR_ALIGN">
     <type>integer</type>
     <valid_values></valid_values>
     <default_value>1</default_value>
     <values match="last">
-      <value compset="1850_DATM%CPLHIST">1</value>
+      <value compset="1850_DATM%CPLHIST">-999</value>
     </values>
     <group>run_component_datm</group>
     <file>env_run.xml</file>
-    <desc>Simulation year corresponding to starting year (only used for CPLHIST3HrWx mode)</desc>
+    <desc>Simulation year corresponding to starting year (only used when DATM_MODE is CPLHIST)</desc>
```

Comment:
Since the user now needs to set this manually, I think the documentation should be expanded. Here's my suggestion, but I'd like you to confirm this, because I'm not confident of my understanding of this variable:

```xml
    <desc>Simulation year corresponding to DATM_CPLHIST_YR_START. This will usually be the same as the year in RUN_STARTDATE, but it can be set to a different year to start the simulation with a different year of forcing data. (Only used when DATM_MODE is CPLHIST.)</desc>
```

Whatever documentation we add for this one, the same documentation should be added for DLND and DROF CPLHIST_YR_ALIGN.

Reply:
I'm not sure this is right. Say you had an 1850 simulation and you were writing coupler history output for years 220-230. Now you wanted to use that output to spin-up the ocean. You now start from year 1 in the ocean - but cycle over years 220-230. I think it will be more confusing to add this documentation. Frankly, I think that cplhist mode is really geared to more expert users. We could give some examples in the documentation for clarity - but I think that would be better in the user's guide as opposed to the xml file.

Reply:
My understanding of the align year is that, in order to do what you're suggesting, you would set the align year to 1, which I think is consistent with my suggested wording. And I do still think that, for spinup runs, the typical operation would be to set the align year to the year in RUN_STARTDATE... and if that is correct (we could ask Keith L & Keith O to confirm), I do feel it would be helpful to add a brief sentence to that effect so that people don't have to dig all through the user's guide to see how to set this unintuitive value.

But I'll let that go if you still disagree :-) - in which case, I'm fine with putting additional documentation in the user's guide instead of the xml file, as long as it appears somewhere. Your idea of providing some examples of typical use cases sounds like a good one, regardless of whether this extra documentation appears in the xm file. (And I'm fine with that being deferred to a separate issue/pr - in that case we should open an issue for it.)